apps/balancer-tools/components.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
what do we need this file for?

Reply:
This is created on shadcn's init

Reply:
https://ui.shadcn.com/docs/installation/next

Reply:
that's a new one for me, I'd just copy the components themselves 😅